src/Tizen.Network.Tethering/Tizen.Network.Tethering/TetheringExtClient.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+
+namespace Tizen.Network.Tethering
+{
+    /// <summary>
+    /// This class contains information related to the connected/disconnected client.
+    /// </summary>
+    /// <since_tizen> 13 </since_tizen>
+    public class TetheringExtClient : IDisposable
+    {
+        IntPtr _client;
+        private string _ipAddr;
+        private string _macAddr;
+        private string _hostname;
+        private bool _disposed = false;
+
+        internal TetheringExtClient(IntPtr client)
+        {
+            IntPtr clonedClient;
+            int ret = Interop.TetheringExt.CloneClient(out clonedClient, client);
+            if (ret != (int)TetheringError.None) {
+                Log.Error(Globals.LogTag, "Error cloning tethering ext client: " + (TetheringError)ret);
+            }
+            _client = clonedClient;
+        }
+
+        /// <summary>
+        /// Destructor function for the class.
+        /// </summary>
+        /// <since_tizen> 13 </since_tizen>
+        ~TetheringExtClient()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// This function destroys the current instance of the class.
```

Comment:
this is not precise, it rather releases (unmanaged) resources, performs kind of clean up